lib/ex_unit/test/ex_unit/capture_io_test.exs

```diff
@@ -1,7 +1,7 @@
 Code.require_file("../test_helper.exs", __DIR__)
 
 defmodule ExUnit.CaptureIOTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: true
```

Comment:
I would not make this test in particular async because we want to be more precise on this one:

```suggestion
  use ExUnit.Case
```